rq/job.py

```diff
@@ -384,6 +385,21 @@ def set_id(self, value):
             raise TypeError('id must be a string, not {0}'.format(type(value)))
         self._id = value
 
+    def set_heartbeat(self, heartbeat, pipeline=None):
```

Comment:
Can we also rename this to `heartbeat()` so that it's consistent with `worker.heartbeat()`?